<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>curveProperties</class>
 <widget class="QWidget" name="curveProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Curve Properties</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabCreateChild">
    <attribute name="title">
     <string>Create Child Curve</string>
    </attribute>
    <widget class="QComboBox" name="cmbPlotType">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>1D Plot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2D Plot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Real FFT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Complex FFT</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lblPlotType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot Type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbXAxisSrc">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblXAxisSrc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X Axis Source</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbYAxisSrc">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblYAxisSrc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y Axis Source</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblDestPlotName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot Name</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbDestPlotName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblDestCurveName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Curve Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDestCurveName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabCurveMath">
    <attribute name="title">
     <string>Curve Math</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="cmdApply">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdDone">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
